_includes/meta.html

```diff
@@ -1,14 +1,8 @@
-{% comment %} 
-You shouldn't have to edit this page very often. But if you want to modify some of the meta-data for the 
+{% comment %}
+You shouldn't have to edit this page very often. But if you want to modify some of the meta-data for the
 site, this is the place to do it.
 {% endcomment %}
-<!DOCTYPE html>
-<!--[if lt IE 9]>
-<html class="lt-ie9">
-<![endif]-->
-<!--[if gt IE 8]><!-->
-<html lang="en">
-<!--<![endif]-->
```

Comment:
Moved the `html` tag to the "default" layout and removed the comments for IE, we don't need to support below IE11